dwitter/migrations/0013_auto_20170226_1249.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('dwitter', '0012_auto_20170225_2056'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Applying the migration unlinks all dweets:

<img width="608" alt="screenshot 2017-02-28 08 50 53" src="https://cloud.githubusercontent.com/assets/610925/23398068/094d61f6-fd93-11e6-9054-d2975c94e9fc.png">